examples/threathunter/threat_intelligence/requirements.txt

```diff
@@ -7,3 +7,4 @@ urllib3>=1.24.2
 cbapi>=1.5.6
 python_dateutil==2.8.1
 PyYAML==5.1.2
+schema
```

Comment:
If you want this to be required the package then it needs to be included in `setup.py`

Reply:
The README mentions using `pip3 install -r requirements.txt`. Do you think it's necessary to modify the `setup.py` file to include schema?

Reply:
I think requirements is enough then since they are not installing from PyPI instead they are cloning and installing from the repo.